### PR TITLE
Get exercises controller fix

### PR DESCRIPTION
**Pull request by AntonZinchenko32**
Adds swagger for adding and deleting exercises

### CODE REVIEW

File: controller/diary/exercises/deleteExerciseById.js [hunk 1]
```diff
@@ -31,6 +29,8 @@ const deleteExerciseById = async (req, res) => {
     { new: true }
   );
 
-  res.status(200).json(data);
+  res
+    .status(200)
```
Comment (Vitalii-Malyk) [inline controller/diary/exercises/deleteExerciseById.js]: We don’t set the response status to 200, it’s the default

File: controller/diary/exercises/getExercisesByDate.js [hunk 2]
```diff
@@ -4,11 +4,11 @@ const getExercisesByDate = async (req, res) => {
...
   const { date: receivedDate } = req.body;
 
-  const result = await DiaryExercise.findOne({
+  let result = await DiaryExercise.findOne({
     ownerId: owner,
     date: receivedDate,
   });
 
-  res.status(200).json(result);
+  res.status(200).json({ data: result });
```
Comment (Vitalii-Malyk) [inline controller/diary/exercises/getExercisesByDate.js]: We don’t set the response status to 200, it’s the default

**Review by Vitalii-Malyk — COMMENTED** [review]
great job

File: controller/diary/exercises/deleteExerciseById.js (hunk 1 above)
Comment (AntonZinchenko32) [reply controller/diary/exercises/deleteExerciseById.js]: fixed

File: controller/diary/exercises/getExercisesByDate.js (hunk 2 above)
Comment (AntonZinchenko32) [reply controller/diary/exercises/getExercisesByDate.js]: fixed

**Review by Ksuyaresko — APPROVED** [review]
Good job 👍

**Pull request merged**